src/Options/OptionsInterface.php

```diff
@@ -23,6 +23,7 @@ interface OptionsInterface {
 	public const CONTACT_INFO_SETUP                        = 'contact_info_setup';
 	public const DELAYED_ACTIVATE                          = 'delayed_activate';
 	public const DB_VERSION                                = 'db_version';
+	public const ENHANCED_CONVERSION_STATUS                = 'enhanced_conversion_status';
```

Comment:
Here as well does it make sense to prefix it with ADS_ so it's together with all the other Ads settings?